src/app/technical-record-search/technical-record-search.component.html

```diff
@@ -16,36 +16,19 @@ <h1 class="govuk-heading-xl">Search for a technical record</h1>
             </label>
             <span *ngIf="searchError.length" id="name-error" class="govuk-error-message">
               <span class="govuk-visually-hidden">Error:</span>
-
-              <div *ngIf="searchError.includes('No resources match the search criteria.')">
-                Vehicle not found, check the vehicle registration mark, trailer ID, vehicle
-                identification number or change <br />
-                the search criteria to find a vehicle
-              </div>
-
-              <div *ngIf="searchError.length">
-                Enter a vehicle registration mark, trailer ID or vehicle identification number
-              </div>
-
-              <div
-                *ngIf="
-                  searchError.includes('The provided partial VIN returned more than one match.')
-                "
-              >
-                Multiple vehicles found, search using the full vehicle identification number
-              </div>
+              <p class="white-space--normal">{{ searchError }}</p>
```

Comment:
who do we need white space. What are you trying to achieve. Do you need this `&nbsp;`

Reply:
![image](https://user-images.githubusercontent.com/55185857/82084024-4fe74f80-96f3-11ea-8212-a8999addadcf.png)